src/runtime_config/gov_root_ca_certs.lua

```diff
@@ -0,0 +1,126 @@
+-- Copyright (c) Microsoft Corporation. All rights reserved.
+-- Licensed under the Apache 2.0 License.
+
+-- This file defines the default initial contents (ie, Lua scripts) of the governance scripts table.
+
+return {
+  pass = [[
+  tables, calls, votes = ...
+  -- interface definitions
+  PASSED = 1
+  PENDING = 0
+  REJECTED = -1
+  STATE_ACTIVE = "ACTIVE"
+
+  -- count member votes
+  member_votes = 0
+
+  for member, vote in pairs(votes) do
+    if vote then
+      member_votes = member_votes + 1
+    end
+  end
+
+  -- count active members
+  members_active = 0
+
+  tables["ccf.members"]:foreach(function(member, details)
+    if details["status"] == STATE_ACTIVE then
+      members_active = members_active + 1
+    end
+  end)
+
+  -- check for raw_puts to sensitive tables
+  SENSITIVE_TABLES = {"ccf.whitelists", "ccf.governance.scripts"}
+  for _, call in pairs(calls) do
+    if call.func == "raw_puts" then
+      for _, sensitive_table in pairs(SENSITIVE_TABLES) do
+        if call.args[sensitive_table] then
+          -- require unanimity
+          if member_votes == members_active then
+            return PASSED
+          else
+            return PENDING
+          end
+        end
+      end
+    end
+  end
+
+  -- validate update_root_ca_cert calls
+  for _, call in pairs(calls) do
+    if call.func == "update_root_ca_cert" then
+      cert_der = pem_to_der(call.args.cert)
+      claims = verify_cert_and_get_claims(cert_der)
+      expected_mrsigner = "ca9ad7331448980aa28890ce73e433638377f179ab4456b2fe237193193a8d0a"
```

Comment:
Could acceptable mrsigners also be stored in a KV table, modified by governance proposals?

Reply:
I guess so, not sure if there's any advantage to it. @achamayou ?

Reply:
They could be, the only benefit is you could do a more limited proposal to change them rather than change the whole constitution if they need updating. I think it's overkill for now, but we can do it easily enough if/when necessary.